plugin/src/test/java/org/wso2/carbon/uuf/maven/bean/AppConfigTest.java

```diff
@@ -60,6 +60,28 @@ public void testLoginPageUriValidations() {
     }
 
     @Test
+    public void testSessionManagerValidations() {
+        AppConfig appConfig = createAppConfig();
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager(""));
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager("class"));
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager("c/lass"));
+        Assert.assertThrows(IllegalArgumentException.class, () -> appConfig.setSessionManager("c*lass"));
+
+        // Null class name
+        appConfig.setSessionManager(null);
+        Assert.assertEquals(appConfig.getSessionManager(), null);
+
+        // Valid class names
+        String className = "c.lass";
```

Comment:
I don't see a point of having this test scenario as the next test scenario  features a valid, correct class name.